### ISSUE TITLE
Add resources from other outlets

### ISSUE THREAD

**Issue by whoabuddy** [stacksgov/resources#33]
Over time there have been some great suggestions from both the forum and in Discord. I did my best to capture those here, and with the wealth of links we have, it might prompt some content reorganization. Thank you everyone for contributing, and if I missed something, feel free to comment!

@blockstackers
- https://blog.cosmos.network/cosmos-governance-february-update-8563c7784a43

@lrettig 
- https://medium.com/@lex_node/defining-decentralization-for-law-58ca54e18b2a
- https://www.eventbrite.com/e/radicalxchange-annual-conference-2020-tickets-102439533402
- https://www.etherean.org/blockchain/economics/society/2020/05/02/to-share-or-not-to-share.html

@joberding 
- https://youtu.be/-lPj8fFkT4c
- https://forum.blockstack.org/t/governance-working-group-update-april-29-2020/10754
- https://forum.blockstack.org/t/participate-in-blockstack-governance-take-the-stacks-governance-survey/10742
- https://onlinelibrary.wiley.com/doi/full/10.1111/j.1083-6101.1996.tb00186.x

@blocks8
- https://corda.network/

Kieran
- https://kieranalexis.dpage.io/digital-baseline-the-flat-plain
- https://kieranalexis.dpage.io/blockstack-governance-wg-side-notes
- https://nakamoto.substack.com/
- https://podcasts.apple.com/us/podcast/episode-71-crossing-river-ethereum-governance-hudson/id1307284590?i=1000473340547
- https://www.youtube.com/watch?v=LNTsQsm6B44

**Comment by whoabuddy** [comment]
From #37, Matt K: https://blockcommons.red/crypto-governance-research/overviews

**Comment by whoabuddy** [comment]
Three more from @lrettig on Discord:
- https://medium.com/@Melt_Dem/the-presidio-principles-fdf40ed40805
- https://www.coindesk.com/makerdaos-problems-are-a-textbook-case-of-governance-failure
- https://near.org/blog/introducing-the-near-foundation/

**Comment by lrettig** [comment]
> Three more from @lrettig on Discord:
> 
> * https://medium.com/@Melt_Dem/the-presidio-principles-fdf40ed40805
> * https://www.coindesk.com/makerdaos-problems-are-a-textbook-case-of-governance-failure
> * https://near.org/blog/introducing-the-near-foundation/

Thanks for grabbing these. The presidio principles github repo can be linked here for sure (https://github.com/WEF-Blockchain/Presidio-Principles/blob/master/presidio-principles.md). And the stuff on Maker is helpful background reading. I don't think we need to link the article introducing NEAR Foundation, but we could link some documentation on the foundation under https://github.com/stacksgov/resources#examples.

**Comment by whoabuddy** [comment]
Given the wealth of content, I feel like we can reorganize things a bit, I'll take a fresh look once the notes are all caught up and submit a PR with what I think :+1:

**Comment by whoabuddy** [comment]
Ubuntu is another large and long-lived project, with their own community standards and governance:

https://ubuntu.com/community
https://ubuntu.com/community/governance

I feel like each of these items deserves a short explanation, summary, or some way to say "this is why this item is on the list", even if its just for the general category (i.e. "Open Source Governance").

Yes, everyone should read all the resources, but in case they don't, knowing the motivation and/or gist of each would be helpful in understanding where to start. The list is very comprehensive which could easily be perceived as intimidating as well.

**Comment by lrettig** [comment]
Agree. Some basic context/interpretation could be very helpful. I'm happy to fill that in for Ethereum :)

**Comment by Topper88** [comment]
I find telegram's native discussbot a near perfect match to keep discussions
- in one channel 
- on a timeline
- separate by topic
example: https://t.me/talk_stacks
I opened this channel originally for noting and I can only recommend to have similar for stacks.gov or I can simply give admin permissions

**Comment by lrettig** [comment]
@Topper88 very interesting! I had never seen Telegram used that way before. Thanks for sharing. I have to give some thought to where this fits in relative to existing channels such as Discord, Discourse, Github, etc. :)

**Comment by whoabuddy** [comment]
Grin has some interesting examples of their governance structure:
https://grin.mw/governance
https://github.com/mimblewimble/grin-rfcs

Corporate Governance for Complex Cryptocurrencies
https://www.law.ox.ac.uk/business-law-blog/blog/2017/08/corporate-governance-complex-cryptocurrencies

**Comment by lrettig** [comment]
Some fantastic resources I've found as part of the RadicalxChange event today:

- http://digitaljuries.com/
- https://communityrule.info/create/
- https://ethicalsource.dev/definition/
- https://democraticmediums.info/
